src/test/pretty/unary-op-disambig.rs

```diff
@@ -16,4 +16,4 @@ fn alt_semi() -> isize { match true { true => { f() } _ => { } }; -1 }
 
 fn alt_no_semi() -> isize { (match true { true => { 0 } _ => { 1 } }) - 1 }
 
-fn stmt() { { f() }; -1; }
+fn stmt() { { f() }; let _ = -1; }
```

Comment:
Isn't this _not_ testing what the test is trying to test? I don't see this ever regressing, but it is something to keep in mind.